src/add/operator/combineLatest.ts

```diff
@@ -5,6 +5,6 @@
 import {Observable} from '../../Observable';
 import {combineLatest} from '../../operator/combineLatest';
 
-Observable.prototype.combineLatest = combineLatest;
+Observable.prototype.combineLatest = <any>combineLatest;
```

Comment:
this patch operator's now auto generated, so instead of changing this script need to be updated - is that correct?